float64 ArmVelDes_1
float64 ArmVelDes_2
float64 ArmVelDes_3
float64 ArmVelDes_4
float64 ArmVelDes_5